keras_cv/layers/preprocessing/color_jitter.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class ColorJitter(layers.Layer):
```

Comment:
Lets subclass `BaseImageAugmentationLayer`

Reply:
https://github.com/keras-team/keras-cv/blob/master/.github/API_DESIGN.md#baseimageaugmentationlayer

Reply:
@LukeWood I already did that on my end, didn't push a new commit yet. As it was suggested (qlzh) [HERE](https://github.com/keras-team/keras-cv/pull/158#discussion_r826199615), I needed to use `tf.image.adjust_brightness` instead of `tf.image.random_brightness` and others. Also, needed some clarification on the [ISSUE](https://github.com/tensorflow/tensorflow/issues/55374). Also on [THIS](https://github.com/keras-team/keras-cv/issues/209#issuecomment-1079237019). But now, I will follow [THIS](https://github.com/keras-team/keras-cv/pull/158#discussion_r836150048), if I understand correctly.

Reply:
Would you be willing to push the new version so we can see it?  It will help in guiding the decision.  Thanks!

Reply:
Done.